katello.gemspec

```diff
@@ -43,6 +43,9 @@ Gem::Specification.new do |gem|
   gem.add_dependency "runcible", ">= 2.11.0", "< 3.0.0"
   gem.add_dependency "anemone"
   gem.add_dependency "zest"
```

Comment:
we should remove the zest dependency here, right?

Reply:
Seems reasonable to me.